cmd/microcloud/join.go

```diff
@@ -42,6 +42,12 @@ func (c *cmdJoin) Run(cmd *cobra.Command, args []string) error {
 		return cmd.Help()
 	}
 
+	fmt.Println("Waiting for services to start ...")
```

Comment:
Will this now ever reach the code down below?

```go
if status.Ready {
	return fmt.Errorf("MicroCloud is already initialized, run 'microcloud add' instead")
}
```

Reply:
True, I'll remove it.

Reply:
Fixed.